docs/source/model_design/concept_model/standard_results_formats/index.rst

```diff
@@ -0,0 +1,197 @@
+
+- Shared language and classification of terms may aid in communication between researchers and engineers
+
+Formatting guide
+-----------------
+
+General considerations
+++++++++++++++++++++++
+
+1. Consistency with GBD names is preferred over internal/other types of consistency. Generally, GBD names (such as age_group_name, location_name, cause_name, etc.) should appear in Vivarium results as :code:`{gbd_name}.replace(" ","_").lower()`. Note that sometimes GBD names things in inconsistent ways. For instance, "6-11 months" and "12 to 23 months". However, this will allow us to easily merge Vivarium results and GBD data using their shared names.
+
+2. Successful execution of these standard formats depends on researcher documentation of requested model outputs within the simulation concept model document.
+
+.. todo::
+
+  Link to updated template doc when ready
+
+3. If a researcher-requested output does not clearly fit within the examples outlined below, researchers and engineers should meet to determine desired results formatting and document the format within the project concept model requested outputs section.
+
+Parameters in results dataframes
+++++++++++++++++++++++++++++++++
+
+.. list-table:: Parameters
+  :header-rows: 1
+
+  * - Parameter
+    - Value type
+    - Definition
+    - Possible values
+  * - :code:`input_draw`
+    - integer
+    - Simulated input draw number
+    - [0, 999]
+  * - :code:`scenario`
+    - string
+    - Simulated scenario name
+    - * baseline
+      * alternative
+      * etc.
+  * - :code:`age`
+    - string
+    - Age group of stratum
+    - :code:`gbd_age_group_name.replace(" ","_").lower()`
+      
+      * early_neonatal
+      * 50_to_54
+      * etc.
+  * - :code:`sex`
+    - string
+    - Sex of stratum
+    - * male
+      * female
+  * - :code:`year`
+    - integer
+    - Simulated year
+    - * 2022 
+      * 2023
+      * etc.
+  * - :code:`{other_stratifying_variables}` (example: :code:`treatment`)
+    - string
+    - Some other variable by which to stratify values (in the same way as age, sex, year, etc.). Intervention coverage is a common example.
+    - Name and values of {other_stratifying_variables} to be defined in advance of implementation by researchers.
+
+
+      Example:
+
+      * covered
+      * uncovered
+  * - :code:`measure`
+    - string
+    - Description of what is being measured about the entity
+    - * ylds
+      * ylls
+      * deaths
+      * person_time
+      * transition_count
+      * first_moment
+      * second_moment
+  * - :code:`entity_type`
+    - string
+    - Type of entity being measured
+    - * cause
+      * rei
+      * {other}
+  * - :code:`entity`
+    - string
+    - Name of entity being measured
+    - :code:`gbd_{entity_type}_name.replace(" ","_").lower()`
+      
+      * measles
+      * child_stunting
+  * - :code:`sub_entity`
+    - string
+    - Name of entity state or category
+    - * measles/susceptible_to_measles
```

Comment:
Is there ever a matching GBD entity here? e.g. a sequela?

Reply:
Hmm, I think it's possible but not common. One issue is that we'll pretty much always have some sort of susceptible state that won't be included in the GBD sequelae. For many causes I think sequelae won't really be relevant and applies more to symptomology than stage of disease. 

There are cases were we might model sequelae as cause model states (maybe for cancers like shown in the diagram below?), but I think it will probably be rare and the names are typically wordy enough that we might want to rename them anyway. 
![image](https://user-images.githubusercontent.com/54918472/197897558-d5066eee-7909-4040-bc40-a883cc081f4a.png)

One thing I did think through though was whether it would be preferable to have either...

A: entity=measles, sub_entities=measles, susceptible_to_measles, recovered_from_measles 

or 

B: entity=measles, subentities=infected, susceptible, recovered

Our current results are structured like A, but that's really because we don't break apart entities and sub-entities. I might update it to B unless you think otherwise!

Reply:
Makes sense. In MM there was kind of a mapping between sequelae and cause states but it wasn't 1:1. And, great point that we wouldn't want to use the sequela names anyway, because they are so cumbersome.

I like B!